es/07_1_Creando_una_Transaccion_Bitcoin_Parcialmente_Firmada.md

```diff
@@ -500,10 +500,10 @@ Finalmente, si en cambio necesita más control y elige usar `utxoupdatepsbt` (qu
 
 ![](images/psbt-roles-for-cli-3.png)
```

Comment:
- `![](images/psbt-roles-for-cli-3.png)` should probably be `![](../images/psbt-roles-for-cli-3.png)`.